viadot/utils.py

```diff
@@ -396,5 +397,25 @@ def call_shell(command):
 
 
 def df_snakecase_column_names(df: pd.DataFrame) -> pd.DataFrame:
-    df.columns = df.columns.str.strip().str.replace(" ", "_").str.replace("-", "_").str.lower()
+    df.columns = (
+        df.columns.str.strip().str.replace(" ", "_").str.replace("-", "_").str.lower()
+    )
     return df
+
+
+def add_viadot_source_column(func):
+    """Decorator to be added to 'to_df' methods of sources.
+    Adds new '_viadot_source' column to the resulting df
+    indicating the name of the souce the df is coming from"""
```

Comment:
I'd keep the quotes on separate lines, also you have a typo ("source").

Reply:
Modified